en/drafts/originals/agent-based-model-communication-networks.md

```diff
@@ -392,35 +402,32 @@ plt.ylabel("Number of Agents")
+Can you rewrite the function above to plot the number of sent letters? What histogram do you expect for that? *Hint*: Are agents sending letters in every round?
 
-#### 2.5.5 Adding space
+#### 2.5.5 Adding Space
 
-Many ABMs have a spatial element, with agents moving around and interacting with neighbors. `Mesa` currently supports two overall kinds of spaces:
-`grid`, and `continuous`. Grids are divided into cells, and agents can only be on a particular cell, like pieces on a chess board. Continuous space, in contrast, allows agents to have any arbitrary position. Both grids and continuous spaces are frequently toroidal, meaning that the edges of this 'world' wrap around, with cells on the right edge connected to those on the left edge, and the top to the bottom. This prevents some cells having fewer neighbors than others, or agents being able to go off the edge of the environment.
+Let's add some more realistic behavior by introducing space between the agents, which should influence their letter-sending decisions. `Mesa` currently supports two kinds of spaces: `grid` and `continuous`. Grids are divided into cells, and agents exist on one particular cell, like pieces on a chess board. Continuous space, in contrast, allows agents to take any arbitrary position. Both grids and continuous spaces are frequently [toroidal](https://en.wikipedia.org/wiki/Toroid), meaning that the edges of this 'world' wrap around, with cells on the right edge connected to those on the left edge, and the top to the bottom. This prevents some cells having fewer neighbors than others, or agents being able to fall off the edge of the environment.
 
-Let’s add a simple spatial element to our model by putting our agents on a grid and make them walk around at random. Instead of sending a letter to any random agent, they’ll give it to an agent on the same cell. We could imagine that this represents them being close enough to know of one another and have reason to send a letter in the first place.
+Let’s add a simple spatial element to our model by putting our agents on a grid, and making them move around it at random. Instead of sending a letter to any random agent, they’ll give it to an agent on the same cell. We could imagine that this represents them existing close enough to know each other and have reasons to send a letter in the first place.
 
-`Mesa` has two main types of grids: `SingleGrid` and `MultiGrid`[^22]. `SingleGrid` enforces at most one agent per cell; `MultiGrid` allows multiple agents to be in the same cell. Since we want agents to be able to share a cell, we use `MultiGrid`.
+`Mesa` offers two main types of grids: `SingleGrid` and `MultiGrid`.[^22] `SingleGrid` enforces at most one agent per cell; `MultiGrid` allows multiple agents in the same cell. Since we want agents to be able to share a cell, we use `MultiGrid`, which we instatiate with width and height paramaters, and which will always be toroidal:
```

Comment:
It's about the possibility of width and height, so as arguments of the model function, see §98 

Regarding toroida spacel: exactly! I'd suggest "...and which will always be toroidal (hence the 'true' designation)"

Reply:
```suggestion
`Mesa` offers two main types of grids: `SingleGrid` and `MultiGrid`.[^22] `SingleGrid` enforces at most one agent per cell; `MultiGrid` allows multiple agents in the same cell. Since we want agents to be able to share a cell, we use `MultiGrid`, which we instatiate with width and height paramaters, and which will always be toroidal (hence the 'true' designation):
```